camayoc/tests/qpc/cli/csv_report_parsing.py

```diff
@@ -0,0 +1,135 @@
+# coding=utf-8
+"""Helper functions for parsing csv reports."""
+
+import csv
+
+EXPECTED_SUMMARY_REPORT_ID_FIELDS = [
+    'Report ID',
+    'Report Type',
+    'Report Version',
+    'Report Platform ID',
+]
+
+EXPECTED_DETAIL_REPORT_ID_FIELDS = [
```

Comment:
If you convert above to be a tuple then you could make this something like:

```python
EXPECTED_DETAIL_REPORT_ID_FIELDS = EXPECTED_SUMMARY_REPORT_ID_FIELDS + (
    'Number Sources',
)

Reply:
I like that :)